pkg/plugin/loader/loader_test.go

```diff
@@ -192,3 +192,24 @@ func TestFilterList(t *testing.T) {
 		t.Errorf("expected %+#v, got %+#v", expected, filtered)
 	}
 }
+
+func TestLoadAllPlugins(t *testing.T) {
+	namespace := ""
+	sonobuoyImage := "whatever"
+	imagePullPolicy := "whatever"
+	searchPath := []string{path.Join("testdata", "plugin.d"), path.Join("testdata", "plugin.d")}
+	selections := []plugin.Selection{
+		plugin.Selection{Name: "daemonset"},
+		plugin.Selection{Name: "job"},
+	}
+	plugins, err := LoadAllPlugins(namespace, sonobuoyImage, imagePullPolicy, searchPath, selections)
```

Comment:
whoops this test is broken. it passes but only because no tests are run. Fixing.